doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-commands-samples-ref.adoc

```diff
@@ -0,0 +1,876 @@
+            "poll-interval": null
+          },
+          "config-items": [
+
+          ],
+          "container-alias": null
+        }
+      }
+    }
+  ]
+}
+----
+--
+ListContainersCommand::
++
+--
+Returns a list of KIE containers that have been created in the {KIE_SERVER}.
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|
+|
+|Required
+
+|
+|
+|Required
+
+|
+|
+|Optional
+
+|
+|
+|Optional
+|===
+
+.Example Java command
+[source,java]
+----
+
+----
+
+.Example REST request body (JSON)
+[source,json]
+----
+{
+  "commands" : [ {
+    "list-containers" : {
+      "kie-container-filter" : {
+        "release-id-filter" : { },
+        "container-status-filter" : {
+          "accepted-status" : [ "CREATING", "STARTED", "FAILED", "DISPOSING", "STOPPED" ]
+        }
+      }
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "List of created containers",
+      "result": {
+        "kie-containers": {
+          "kie-container": [
+            {
+              "container-id": "command-script-container",
+              "release-id": {
+                "group-id": "com.redhat",
+                "artifact-id": "Project1",
+                "version": "1.0"
+              },
+              "resolved-release-id": {
+                "group-id": "com.redhat",
+                "artifact-id": "Project1",
+                "version": "1.0"
+              },
+              "status": "STARTED",
+              "scanner": {
+                "status": "STARTED",
+                "poll-interval": 5000
+              },
+              "config-items": [
+                {
+                  "itemName": "RuntimeStrategy",
+                  "itemValue": "SINGLETON",
+                  "itemType": "java.lang.String"
+                },
+                {
+                  "itemName": "MergeMode",
+                  "itemValue": "MERGE_COLLECTIONS",
+                  "itemType": "java.lang.String"
+                },
+                {
+                  "itemName": "KBase",
+                  "itemValue": "",
+                  "itemType": "java.lang.String"
+                },
+                {
+                  "itemName": "KSession",
+                  "itemValue": "",
+                  "itemType": "java.lang.String"
+                }
+              ],
+              "messages": [
+                {
+                  "severity": "INFO",
+                  "timestamp": {
+                    "java.util.Date": 1538504619749
+                  },
+                  "content": [
+                    "Container command-script-container successfully created with module com.redhat:Project1:1.0."
+                  ]
+                }
+              ],
+              "container-alias": null
+            }
+          ]
+        }
+      }
+    }
+  ]
+}
+----
+--
+CallContainerCommand::
```

Comment:
two mandatory attributes - container id of type string and payload which represents marshalled all other commands to be invoked on the container identified by container id

Reply:
Done.